### PR TITLE
Adding MoMs to main

### PR DESCRIPTION
**Pull request by DT-DawidWroblewski**
#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:

merge MoMs into main

### CODE REVIEW

**Review by bigludo7 — DISMISSED** [review]
OK !

File: code/API_definitions/checkSimSwap.camara.swagger.yaml
Comment (fernandopradocabrillo) [inline code/API_definitions/checkSimSwap.camara.swagger.yaml:1]: This should not be merge, it is an old specification

File: documentation/API_documentation/Check_sim_swap_API.md
Comment (fernandopradocabrillo) [inline documentation/API_documentation/Check_sim_swap_API.md:1]: This one too, it is an old one

**Review by fernandopradocabrillo — COMMENTED** [review]
@DT-DawidWroblewski 
I see a couple files that should be deleted from the PR, they are old ones and deprecated

File: code/API_definitions/checkSimSwap.camara.swagger.yaml
Comment (DT-DawidWroblewski) [reply code/API_definitions/checkSimSwap.camara.swagger.yaml:1]: ok - please commit and delete :)

File: documentation/API_documentation/Check_sim_swap_API.md
Comment (DT-DawidWroblewski) [reply documentation/API_documentation/Check_sim_swap_API.md:1]: ok - please commit and delete :)

Comment (DT-DawidWroblewski) [reply documentation/API_documentation/Check_sim_swap_API.md:1]: @fernandopradocabrillo I fixed it - please review

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM

**Pull request merged**